code/game/objects/items/shields.dm

```diff
@@ -153,3 +153,15 @@
 		slot_flags = null
 		to_chat(user, "<span class='notice'>[src] can now be concealed.</span>")
 	add_fingerprint(user)
+
+/obj/item/shield/makeshift

```

Comment:
Not being a subtype of riot shield means this item won't have no materials, no extra block chance against leap attacks and thrown objects, and be a normal sized item rather than a bulky one, in spite of the description.